cmd/osv-scanner/main_test.go

```diff
@@ -781,28 +780,28 @@ func TestRun_Docker(t *testing.T) {
 	tests := []cliTestCase{
 		{
 			name: "Fake alpine image",
-			args: []string{"", "--docker", "alpine:non-existent-tag"},
+			args: []string{"", "scan", "docker", "alpine:non-existent-tag"},
 			exit: 127,
 		},
 		{
 			name: "Fake image entirely",
-			args: []string{"", "--docker", "this-image-definitely-does-not-exist-abcde"},
+			args: []string{"", "scan", "docker", "this-image-definitely-does-not-exist-abcde"},
 			exit: 127,
 		},
 		// TODO: How to prevent these snapshots from changing constantly
 		{
 			name: "Real empty image",
-			args: []string{"", "--docker", "hello-world"},
+			args: []string{"", "scan", "docker", "hello-world"},
```

Comment:
what if someone passes `osv-scanner scan docker` where `docker` is a valid directory? Similarly for `osv-scanner scan project`. 

Do we need to handle that in a special way, with a warning printed to users?

Reply:
Added a warning message